trees/c-or-cpp/count-and-sum-of-nodes-in-binary-tree.cpp

```diff
@@ -0,0 +1,77 @@
+#include <iostream>
+
+using namespace std;
+
+// Basic Structure of a tree
+class Node
+{
+public:
+    int data;
```

Comment:
## Optional
I think you should create the instance member variable private so it can't be misuse outside of the class and add instance member function to assign address/data  in it